docs/pages/x/api/data-grid/grid-actions-col-def.js

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import MarkdownDocs from '@mui/monorepo/docs/src/modules/components/MarkdownDocs';
+import MarkdownDocs from 'docs/src/modules/components/MarkdownDocs';
```

Comment:
I don't love this change, I feel like the `@mui/...` prefix was a nice way to indicate that it comes from another MUI package. It would be nice if we could have a standardized form to rename packages from the monorepo.

Reply:
I feel like the existing `docs` and `docsx` approach is pretty well established in everyone's head, but I can also second this though, ideally, we should aim at using a similar approach to aliasing/naming all packages and currently `docs`, `docsx` and `typescript-to-proptypes` are probably the only exceptions. 🤔 🤷 
This could become `@mui/docs` or `@mui-internal/docs` (although the former would be a better fit given how the alias names are chosen)

Reply:
> This could become @mui/docs

We already have this one but it's almost empty

I do agree that the current aliases are a mess.
But for me `@mui/monorepo` is not the solution.
I would prefer, as Lukas suggest to move the code on the doc to some `@mui-internal` packages.

Reply:
For me the main pain of `@mui/monorepo` is that in theory anything on the core repo is "public API" because it's used by us, so anytime they break something, maybe we are impacted.

The idea of removing this alias eventually, is to make clear which folders in there repo is public API and which one they can totally rework without caring for others.

So once we are not using `@mui/monorepo` anymore, we can start cleaning the other aliases (move to the new `@mui-internal` nomenclature everywhere would be a good start).

And then we will be able to move those new `@mui-internal` packages to another repo if we want.

Reply:
Ok gotcha. I'm not sure how I feel about the non-required aliases overall, they do require getting used to. When I played with the babel config a few weeks ago I wasn't sure what was what. Non-magic stuff is easier for newcomers, and doesn't require engineers who update the configs to have as much accumulated knowledge of the project.

But I do see the benefits of `@mui-internals`, and if it was all standardized to that I would be reasonably happy.

Feel free to merge anyway, just wanted to voice my concerns.

Reply:
What are you referring to when you talk about "non-required aliases" ?

My dream scenario would be that one day we have one repo for the infra with all the code published on NPM and versioned in well-defined packages.

Reply:
The `@mui/...` to local `./packages` are the required ones:

![image](https://github.com/mui/mui-x/assets/1423607/5eae95aa-5f7a-40ca-937e-cad47d0a3bb1)

Reply:
I'm not following you